scss/civicrm/common/_buttons.scss

```diff
@@ -56,10 +56,11 @@ $button-border-radius: 2px;
     }
   }
 
-  input.crm-form-submit {
+  button.crm-form-submit {
```

Comment:
isn't it possible to completely remove the element here and just leave the class part?

Reply:
this is the same question for every instance of `button.class-name`

Reply:
I tried this initially @reneolivo . But it messes with the css selector order. I mean if I remove `button` then some of the styles are not applied because of order.

Reply:
Just as a thought bubble as well is it worth while keeping input.crm-form-submit and attaching on button.crm-form-submit as another selector for backwards compatibility?

Reply:
that's an interesting point @seamuslee001 , but it needs to be a more involved than that. The new buttons don't need icon alignment, but the old ones do. So we would need to preserve both solutions. I think this would bring an issue in the future because it will be hard to debug why we have these statements that are not targeting "anything" (since we'll probably be checking against latest release).

I think the best approach is to define the target CiviCRM version that Shoreditch will be compatible with. If the user upgraders CiviCRM, they must upgrade Shoreditch too and vice versa.

Reply:
@seamuslee001 @reneolivo 
I also thought about it initially. But then I felt that its not worth it and is going to be more expensive to maintain both kind of buttons. And if someone really needs supports for `input` elements, then they can use older version of Shoreditch.

Reply:
Yeh fair point just thought throw out the question :)